devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -12,15 +12,19 @@
  *
  */
 metadata {
-	definition (name: "Z-Wave Dimmer Switch Generic", namespace: "smartthings", author: "SmartThings") {
+	definition (name: "Z-Wave Dimmer Switch Generic", namespace: "smartthings", author: "SmartThings", category: "C5") {
 		capability "Switch Level"
 		capability "Actuator"
 		capability "Switch"
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
+		capability "Configuration"
+		capability "Health Check"
 
 		fingerprint inClusters: "0x26", deviceJoinName: "Z-Wave Dimmer"
+		fingerprint mfr:"0063", prod:"4944", deviceJoinName: "Z-Wave In-Wall Dimmer"
```

Comment:
Hi @juano2310 - 

This work is to add Device Health capability and support to Z-Wave dimmers. 

Can you elaborate on the difference between this DTH and `dimmer-switch.groovy`? 
My first impression is for us to change both DTHs. If you can confirm this, we can have Piyush change both `zwave-dimmer-switch-generic.groovy` & `dimmer-switch.groovy`.